release: v1.35.0 (#6868)
GoogleContainerTools/skaffold: CHANGELOG.md

# v1.35.0 Release - 11/16/2021
**Linux amd64**
`curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v1.35.0/skaffold-linux-amd64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin`

**Linux arm64**
`curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v1.35.0/skaffold-linux-arm64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin`

**macOS amd64**
`curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v1.35.0/skaffold-darwin-amd64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin`

**macOS arm64**
`curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v1.35.0/skaffold-darwin-arm64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin`

**Windows**
https://storage.googleapis.com/skaffold/releases/v1.35.0/skaffold-windows-amd64.exe

**Docker image**
`gcr.io/k8s-skaffold/skaffold:v1.35.0`

Note: This release comes with a new config version, `v2beta26`. To upgrade your skaffold.yaml, use `skaffold fix`. If you choose not to upgrade, skaffold will auto-upgrade as best as it can.

Highlights:

> * Skaffold's native support for `ko` builder for golang projects. Please try it out and let us [know](https://skaffold.dev/docs/pipeline-stages/builders/ko/)

Reviewer: ```suggestion
* Skaffold now natively supports building Go projects with [`ko`](https://github.com/google/ko)! Check out the [documentation](https://skaffold.dev/docs/pipeline-stages/builders/ko/) for more information on how it works with Skaffold projects.
```

> * Skaffold performs status-check for stateful sets [#6828](https://github.com/GoogleContainerTools/skaffold/pull/6828)

Reviewer: ```suggestion
* Skaffold now performs status-check for `StatefulSet` resources
```

* docs: add release stage plan to ko builder design doc [#6764](https://github.com/GoogleContainerTools/skaffold/pull/6764)
* docs: Clarify custom local dependencies example [#6827](https://github.com/GoogleContainerTools/skaffold/pull/6827)
* test(ko): Simple integration test for ko builder [#6788](https://github.com/GoogleContainerTools/skaffold/pull/6788)
* test: add integration test for config connector status check [#6839](https://github.com/GoogleContainerTools/skaffold/pull/6839)
* test: fix integration test for stateful-sets [#6829](https://github.com/GoogleContainerTools/skaffold/pull/6829)
* test: update modules testcases [#6813](https://github.com/GoogleContainerTools/skaffold/pull/6813)
* ci: add cancel-workflow-action functionality to all github workflows [#6755](https://github.com/GoogleContainerTools/skaffold/pull/6755)

Huge thanks goes out to all of our contributors for this release:

- Aaron Prindle
- Ahmet Alp Balkan
- Brian de Alwis
- Conor A. Callahan
- Erik Godding Boye
- Gaurav
- Halvard Skogsrud
- Jeremy Lewi
- Marlon Gamez
- Max Brauer
- Nick Kubala
- Pablo Caderno
- Rouan van der Ende
- Tejal Desai

> - dependabot[bot]

Reviewer 2: we don't need to thank dependabot

> - Tejal Desai
> - dependabot[bot]
> - jrcast

Reviewer: ```suggestion
- Tejal Desai
- jrcast
```
